public/app/features/org/SelectOrgPage.tsx

```diff
@@ -34,7 +34,7 @@ export const SelectOrgPage: FC<Props> = ({ setUserOrganization }) => {
   const [orgs, setOrgs] = useState<UserOrg[]>();
 
   const setUserOrg = async (org: UserOrg) => {
-    setUserOrganization(org.orgId);
+    await setUserOrganization(org.orgId);
```

Comment:
Curious how this works. When calling a Thunk you should always wrap the call in `dispatch`. So this would effectively be

```ts
import { useDispatch } from 'react-redux';
...
const dispatch = useDispatch();
const createOrg = async (newOrg: { name: string }) => {
    await dispatch(createOrganization(newOrg));
    window.location.href = getConfig().appSubUrl + '/org'; 
};
```